TemplePlus/gamesystems/gamesystems.cpp

```diff
@@ -791,6 +791,7 @@ void GameSystems::InitializeSystems(LoadingScreen& loadingScreen) {
 	loadingScreen.SetProgress(79 / 79.0f);
 	mPathX = InitializeSystem<PathXSystem>(loadingScreen, mConfig);
 	mPoison = InitializeSystem<PoisonSystem>(loadingScreen);
+	skillSys.Init();

```

Comment:
Why don't you put this inside SkillSystem constructor? Have you tried?

legacysystems.cpp line 577 and on. I'm guessing it should go after the vanilla Init.

Reply:
SkillSys constructor worked fine, thanks!